ci-operator/jobs/infra-build-farm-periodics.yaml

```diff
@@ -502,17 +502,25 @@ periodics:
   decorate: true
   decoration_config:
     timeout: 1h0m0s
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+    workdir: true
   labels:
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-openshift-dptp-4552-clusterpool-pruner
   reporter_config:
     slack:
       channel: '#team-hive-alert'
       job_states_to_report:
+      - success
       - failure
       - error
-      - success
-      report_template: 'Job {{.Spec.Job}}: <{{.Status.URL}}|View logs>'
+      report_template: '{{if eq .Status.State "success"}} :slack-green: Job *{{.Spec.Job}}*
+        succeeded. <{{.Status.URL}}|View logs> <!subteam^team-hive-info>{{else}} :failed:
+        Job *{{.Spec.Job}}* failed with *{{.Status.State}}*. <{{.Status.URL}}|View
```

Comment:
Can we put "View logs" outside of the conditional (so I can see it for success too)?